multi-agent/lib/mas/session/domain/status.py

```
@@ -1,15 +1,23 @@
-    QUEUED = "QUEUED"
-    RUNNING = "RUNNING"
-    COMPLETED = "COMPLETED"
-    FAILED = "FAILED"
-    CANCELLED = "CANCELLED"
-    # Shared-session specific busy statuses:
-    # LOCKED   – session is reserved / queued for execution by another caller
-    # IN_USE   – session is actively being executed by another caller
-    LOCKED = "LOCKED"
-    IN_USE = "IN_USE"
+from enum import Enum
+from typing import FrozenSet
```

Ruff (0.15.18)

[warning] `typing.FrozenSet` is deprecated, use `frozenset` instead

(UP035)